x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_activity_log.tsx

```diff
@@ -71,6 +71,11 @@ export const EndpointActivityLog = memo(
       [hasActiveDateRange, isPagingDisabled, activityLogLoading, activityLogSize]
     );
 
+    const showCallout = useMemo(
```

Comment:
nit: useMemo is unnecessary here:

the value is a primitive (boolean), the calculation is order 1, useMemo is using more memory and CPU on figuring if it should cache the result than the operation would take.

Reply:
I'll improve this in the next PR when there's any change in the UX again 👍